client/dashboard/me/notifications-emails/pause-all-emails/index.tsx

```diff
@@ -0,0 +1,117 @@
+	__experimentalConfirmDialog as ConfirmDialog,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from 'react';
+import { useNotice } from '../hooks/use-notice';
+
+const isAllWpcomEmailsDisabled = ( settings: UserSettings ) => {
+	return settings.subscription_delivery_email_blocked === true;
+};
+
+export const PauseAllEmails = () => {
+	const [ enabled, setEnabled ] = useState( false );
+	const { data: settings } = useSuspenseQuery( userSettingsQuery() );
+	const { createSuccessNotice, createErrorNotice } = useNotice();
+	const queryClient = useQueryClient();
+
+	const { mutate: updateSettings, isPending: isSaving } = useMutation( {
+		...userSettingsMutation(),
+		onSuccess: async () => {
+			const message = enabled ? __( 'All emails paused.' ) : __( 'All emails unpaused.' );
+			createSuccessNotice( message, { type: 'snackbar' } );
+			await queryClient.invalidateQueries( { queryKey: userSettingsQuery().queryKey } );
+		},
+		onError: () => {
+			createErrorNotice( __( 'Failed to save settings.' ), { type: 'snackbar' } );
+		},
+	} );
+	const originalState = isAllWpcomEmailsDisabled( settings );
+	const [ isConfirmDialogOpen, setIsConfirmDialogOpen ] = useState( false );
+
+	useEffect( () => {
+		setEnabled( isAllWpcomEmailsDisabled( settings ) );
+	}, [ settings ] );
+
+	useEffect( () => {
+		if ( isSaving ) {
+			setIsConfirmDialogOpen( false );
+		}
+	}, [ isSaving ] );
+
+	const handleChange = ( checked: boolean ) => {
+		setEnabled( checked );
+	};
+
+	const handleConfirmation = () => {
+		updateSettings( {
+			subscription_delivery_email_blocked: enabled,
+		} );
+	};
+
+	const handleSubmit = ( e: React.FormEvent< HTMLFormElement > ) => {
+		e.preventDefault();
+		if ( enabled ) {
+			setIsConfirmDialogOpen( true );
+		} else {
+			updateSettings( {
+				subscription_delivery_email_blocked: enabled,
+			} );
+		}
+	};
+
+	return (
+		<>
+			<ConfirmDialog
+				onConfirm={ handleConfirmation }
+				onRequestClose={ () => setIsConfirmDialogOpen( false ) }
+				isOpen={ isConfirmDialogOpen }
+				confirmButtonText={ __( 'Yes, I want to pause all emails' ) }
+				cancelButtonText={ __( 'Cancel' ) }
+				title={ __( 'Are you sure you want to pause all emails?' ) }
+				style={ { maxWidth: '480px' } }
+			>
+				<h3>{ __( 'Are you sure you want to pause all emails?' ) }</h3>
+				<p>
+					{ __(
+						"You have active newsletter subscriptions. Pausing emails means you won't receive any newsletter updates."
+					) }
+				</p>
+			</ConfirmDialog>
+			<Card>
+				<CardBody>
+					<form onSubmit={ handleSubmit }>
+						<VStack spacing={ 4 } alignment="start">
+							<CheckboxControl
+								__nextHasNoMarginBottom
+								help={ __(
+									'Pause all email updates from sites you’re subscribed to on WordPress.com. This includes newsletters from the sites you follow. You can turn them back on anytime.'
+								) }
+								label={ __( 'Pause All Emails' ) }
```

Comment:
@lucasmendes-design can confirm, but I believe we've moved away from capitalized case.


```suggestion
								label={ __( 'Pause all emails' ) }
```

Reply:
Updated [132d011](https://github.com/Automattic/wp-calypso/pull/105656/commits/132d01116ea2dcf76327b3843bc4b8cd32ebec2e)